templates/legal/data-privacy/contact.md

```diff
@@ -8,7 +8,7 @@ context:
 <h4 class="p-muted-heading">Version - April 2018</h4>
 <hr style="margin-bottom: 2rem;" />
 
-# Privacy notice - Contact us and enquiries
+# Privacy Notice - Contact us and enquiries
```

Comment:
I'm surprised you feel that "Notices" and "Notice" should be capitalised. I get that "Partner Portal" could be considered a proper noun, but "Privacy notice" doesn't feel like one - especially since it's used in the lower-case form in the below paragraph.

But if you think that's the right call, I'm happy with that.

Reply:
you are right, but I didn't want to change the footer/form notes...